appsignals-tests/images/aws-sdk/aws-sdk-v2/src/main/java/com/amazon/sampleapp/App.java

```diff
@@ -509,7 +530,262 @@ private static void setupS3() {
             var response =
                 faultClient.getObject(
                     GetObjectRequest.builder().bucket("fault-bucket").key("fault-object").build());
-          } catch (Exception e) {
+          } catch (Exception ex) {
+            ex.printStackTrace();
+          }
+          return "";
+        });
+  }
+
+  private static void setupBedrock() {
+    // Localstack does not support Bedrock related services.
+    // We point all Bedrock related request endpoints to the local app,
+    // and then specifically handle each request to return the expected response.
+    // For the full list of services supported by Localstack, see:
+    // https://github.com/testcontainers/testcontainers-java/blob/1f38f0d9604edb9e89fd3b3ee1eff6728e2d1e07/modules/localstack/src/main/java/org/testcontainers/containers/localstack/LocalStackContainer.java#L402
+    var objectMapper = new ObjectMapper();
+    var bedrockClient =
+        BedrockClient.builder()
+            .endpointOverride(URI.create("http://bedrock.test:8080"))
+            .region(Region.US_WEST_2)
+            .credentialsProvider(CREDENTIALS_PROVIDER)
+            .build();
+    var bedrockAgentClient =
+        BedrockAgentClient.builder()
+            .endpointOverride(URI.create("http://bedrock.test:8080"))
+            .region(Region.US_WEST_2)
+            .credentialsProvider(CREDENTIALS_PROVIDER)
+            .build();
+    var bedrockAgentRuntimeClient =
+        BedrockAgentRuntimeClient.builder()
+            .endpointOverride(URI.create("http://bedrock.test:8080"))
+            .region(Region.US_WEST_2)
+            .credentialsProvider(CREDENTIALS_PROVIDER)
+            .build();
+
+    // Response with GetKnowledgeBaseRequest.
+    get(
+        "/knowledgebases/:knowledgeBaseId",
+        (req, res) -> {
+          String knowledgeBaseId = req.params(":knowledgeBaseId");
```

Comment:
Is this output identical to aws-sdk-v1 response? Can we deduplicate and have some shared code? Just a thought.

Reply:
Yes, deduplicated it and move the share code into a shared module in `aws-sdk-base/.../Utils.java`